ingestion/ingest-wordpress.py

```diff
@@ -56,7 +76,42 @@ def post(self):
         do_work()
         return "Post"
 
+class WordpressExportListener(Resource):
+    """
+    `WordpressExportListener` will be the endpoint to be triggered when we want to return something back to a 
+    wordpress site. It requires the following body attributes:
+
+
+    JSON
+    {
+        "id": ID of the source content, 
+        "lang": target language code (ex. "fr"),
+        "content": HTML content (ex. "<p>Bonjour le monde! Je suis Rapi Castillo</p>"),
+        "title": Title of the translated content (ex. "Bonjour le monde!"),
+        "excerpt": Excerpt of the translated content (ex. "Hello this is a french translation xx"),
+        "status": "publish"
+    }
+
+    This will create/update the translated content.
+
+    """
+    def post(self):
+
+        post_data = request.json
+        id=post_data.get("id")
+        lang=post_data.get("lang")
+        message = {
+            "content": post_data.get("content"),
+            "title": post_data.get("title"),
+            "excerpt": post_data.get("excerpt"),
+            "status": post_data.get("status")
+        }
+        
+        return do_export(id, lang, message)
```

Comment:
Rather than receiving the text of the message from a post() parameter, we need to get it from the file system. For instance, here are some examples of spanish (es) output files.
https://github.com/nyc-cto/tms-data/tree/aditya/development/local/source_files/es
I've been using the "wp" prefix to indicate wordpress files. Can you read the json from them?

In our Docker setup, this tms-data git branch gets checked out under /var/tms-data/source_files.

Reply:
Re: actually triggering the code to run -- Ideally we need our Serge wrapper to hit your rest api here. Can you ask Aditya about the best place for the call to happen? If there's no good place to put the hook, we can instead use a celery/cron-ish timer to run your code periodically

Reply:
That sounds good. I'll check in with Aditya.